.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,18 @@
+name: Pull Request
+on:
+  pull_request
+
+jobs:
+  build_image:
+    name: Build Image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Push Docker Image to Github Registry
```

Comment:
ah, I also started pushing to Dockerhub (again) because it turns out you can't pull github docker images unless you authenticate, making the onboarding process a bit more complicated... so for now I'm pushing to both, and using github docker registry for testing (when an image is needed) and dockerhub for lyft/flyte references (as is the case today)

Reply:
Do you have an example of that somewhere? As in, is there an action I can already consume to do the push to dockerhub as well?

Reply:
Nevermind, I found it in flytepropeller. I updated the places we are pushing to also push to dockerhub.